### ISSUE TITLE
ВАЖНО! Необходимо разработать API

### ISSUE THREAD

**Issue by MikhailSklyarov1** [myakininayu/edu-client#22]
Определить структуру JSON, потом разработать API и спроектировать БД (edu-server)

**Comment by MikhailSklyarov1** [comment]
Есть такой вариант JSON
{
  "content": [
    {
      "type": "image",
      "url": "https://example.com/image.jpg",
      "caption": "Картинка 1"
    },
    {
      "type": "video",
      "url": "https://example.com/video.mp4",
      "caption": "Видео 1"
    },
    {
      "type": "text",
      "text": "Текстовый блок 1"
    }
  ]
}

**Comment by myakininayu** [comment]
А может отдельно в каждом типе сделать массив соответствующих элементов? + еще надо помечать, к какому блоку относится информация (Основная, дополнительная, примеры и т.д.)

**Comment by MikhailSklyarov1** [comment]
> 

Да, я тоже сначала так думал. Но как восстановить последовательность? У нас же может быть контент вперемешку. Можно номер указывать, но это парсить потом будет невозможно

**Comment by MikhailSklyarov1** [comment]
А зачем разделение на основную и дополнительную?

**Comment by myakininayu** [comment]
> А зачем разделение на основную и дополнительную?

А как понять, в какую секцию потом закинуть? Запросы отдельные делать?

**Comment by MikhailSklyarov1** [comment]
> > А зачем разделение на основную и дополнительную?
> 
> А как понять, в какую секцию потом закинуть? Запросы отдельные делать?

А зачем разные секции?
